elyra/util/cos.py

```diff
@@ -55,9 +55,10 @@ def __initialize_object_store(self):
         try:
             if not self.client.bucket_exists(self.bucket):
                 self.client.make_bucket(self.bucket)
-        except BucketAlreadyOwnedByYou as ex:
+        except BucketAlreadyOwnedByYou:
+            # Note: the check for `bucket_exists` does not work in the IBM Cos
+            # so we should not re-raise the exception
             self.log.warning("Object Storage bucket already owned by you", exc_info=True)
```

Comment:
This code doesn't make sense to me in the first place.  If we're checking for the bucket's existence (assuming the existence check works), why would we get EITHER of the exceptions `BucketAlreadyOwnedByYou` and `BucketAlreadyExists`??

In my opinion, and given the existence check can't be trusted in the first place, we should call `make_bucket` unconditionally and ignore the two Bucket exceptions.

Reply:
I believe creating a bucket is probably a much heavier operation than checking for its existence, and I don't want to penalize the instances where the check for the bucket exists works. Based on that, I would be more inclined to leave the code as-is.

Reply:
I agree. I figured we'd either leave as is or build a wrapper. I think we should still catch/ignore the exceptions either way.